### DISCUSSION TITLE
Different speed for each book

### DISCUSSION THREAD

**Discussion by lukassnake** [advplyr/audiobookshelf#3633] [Ideas]
Currently, the playback speed setting remains constant across all audiobooks. However, different narrators have varying paces, which means users may need to adjust the speed for each book.

Proposed Enhancement:

**Default Speed Setting:**
Each audiobook should start with a default playback speed of 1x.

**User-Specific Speed Adjustment:**
Allow users to adjust the playback speed according to their preference for each audiobook.

**Persistent Speed Storage:**
The server should store the user's selected playback speed for each individual audiobook.

**Automatic Speed Application:**
When a user selects a different audiobook, the system should automatically apply the stored playback speed for that specific book.
This enhancement will ensure a more personalized and enjoyable listening experience, as users can tailor the playback speed to match the narrator's pace for each audiobook.

Thanks for considering this suggestion and have a nice day.

**Comment by nichwall** [comment]
Duplicate of https://github.com/advplyr/audiobookshelf/issues/1173, https://github.com/advplyr/audiobookshelf/issues/1785, https://github.com/advplyr/audiobookshelf/issues/911
